chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -692,10 +692,18 @@ find-pdf-files-added = { $count ->
 
 select-items-dialog =
     .buttonlabelaccept = Select
+convert-to-standalone = Convert to Standalone
+select-items-convertToStandalone =
+    .label = { convert-to-standalone }
 select-items-convertToStandaloneAttachment =
     .label = { $count ->
-        [one] Convert to Standalone Attachment
-        *[other] Convert to Standalone Attachments
+        [one] { convert-to-standalone } Attachment
+        *[other] { convert-to-standalone } Attachments
+}
+select-items-convertToStandaloneNote =
+    .label = { $count ->
+        [one] { convert-to-standalone } Note
+        *[other] { convert-to-standalone } Notes
```

Comment:
This is another situation where we really can't be obsessed with DRY. We have no way of knowing how "Convert to Standalone" is going to need to be translated in a given language, and there's no reason to expect that the string for "Convert to Standalone" is going to slow into a "Convert to Standalone Attachment" or "Convert to Standalone Note" string.

Reply:
Gotcha! I really need to turn off DRY thinking while dealing with localization. I removed the `{ convert-to-standalone }` and swapped it for the proper "Convert to Standalone" string.